### PR TITLE
fixing issue 5075

### PR DESCRIPTION
**Pull request by codeinnovers**
## Issue Reference

fix for issue #5075
## Description

modified file packages/Webkul/Ui/src/Resources/views/datagrid/table.blade.php to add the number of records in the datagrid and modified packages/Webkul/Admin/src/Resources/lang/en/app.php for translation
## How To Test This?

can be tested on data grids in the admin attached screenshots for the reference
https://nimb.ws/y6TrvD
https://nimb.ws/QBK3wU (filtered grid)
## Documentation
- [ ] My pull request requires an update on the documentation repository.

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/lang/en/app.php [hunk 1]
```diff
@@ -1553,7 +1553,8 @@
             'generate-invoice'                      => 'Automatically generate the invoice after placing an order',
             'set-invoice-status'                    => 'Set the invoice status after creating the invoice to',
             'set-order-status'                      => 'Set the order status after creating the invoice to',
-            'generate-invoice-applicable'           => 'Applicable if automatic generate invoice is enabled'
+            'generate-invoice-applicable'           => 'Applicable if automatic generate invoice is enabled',
+            'records-found'                         => 'Record(s) found'
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/lang/en/app.php:1557]: Please provide all translations present in the lang folder.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/lang/en/app.php (hunk 1 above)
Comment (codeinnovers) [reply packages/Webkul/Admin/src/Resources/lang/en/app.php:1557]: translations pushed for all languages available

**Comment by devansh-webkul** [comment]
Hi, @codeinnovers,

Thanks for the contribution. It's working fine. :rocket: 

There is one more data grid that is fully Ajaxified, can you replicate to that also,
https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue

You can check the orders table for reference.

**Comment by devansh-webkul** [comment]
I am merging this one. You can create a new PR.

**Pull request merged**